lib/sender.js

```diff
@@ -67,67 +68,91 @@ module.exports = {
 */
 function send(client) {
     return function send(message, options) {
-        return new Promise((resolve, reject) => {
-            options = options || {};
+        const body = composeSenderRequestBoddyForOneMessage(message, options);
+        return makeRequest(client, body);
+    };
+}
+
+/**
+    The sendBatch Method for multiple messages
+    @param {Array}  messages - An array of objects containing the fields described in send Method [{message, options}, ... ]
+*/
+function sendBatch(client) {
+    return function sendBatch(messages) {
+        const body = messages.map(m => composeSenderRequestBoddyForOneMessage(m.message, m.options));
+        client.url += "batch";
```

Comment:
must use single quote, @josemigallas 

I just learned that from TravisCI :boom:  :joy: